PlaybackLab/TVOSSwiftSampleApp/TVOSSwiftSampleApp/List/VideoTableViewController.swift

```diff
@@ -0,0 +1,132 @@
+//
+//  ViewController.swift
+//  TVOSSwiftSampleApp
+//
+//  Copyright © 2018 Ooyala. All rights reserved.
+//
+
+import UIKit
+
+class VideoTableViewController: UIViewController, UITableViewDataSource, UITableViewDelegate {
+  
+  var tableOptions : Array<PlayerSelectionOption>!
```

Comment:
I suggest we do this in a _Swift_ style, not in _Obj-C_

1. Try to avoid implicitly unwrapped optionals, it can cause unwanted behavior/errors
2. It's better to omit _Array_ literal if it's a type

It's better to instantiate variables where appropriate during init, so it may look like this
```suggestion
  var tableOptions : [PlayerSelectionOption] = []
```

Reply:
Done